torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -2474,6 +2474,8 @@ class IrParser {
 
     {
       std::array<const char*, kNumViewOps> ViewOps = {
+          "aten::reshape(Tensor(a) self, int[] shape) -> Tensor(a)",
+          "aten::view(Tensor(a) self, int[] size) -> Tensor(a)",
```

Comment:
I thought that we are not fusing `aten::view` since it's not always safe to do so?

Reply:
I thought shape propagation is only done inside graph fusion, so it should have happened after we construct a fusion node within its subgraph, which should only have `view_copy/reshape_copy`. Curious to see how/why we would need to add `reshape/view` into our parser.

On a separate note, putting reshape/view here means we are fusing them anyway. So this would break the alias check we have.

Reply:
I think #1443 fixed the shape propagation issue.